Indicate a minimum rustc version in Cargo.toml?

I failed to compile rinex 0.6.1 and got `error[E0658]: use of unstable library feature 'vec_retain_mut'`.

I then updated Rust with `rustup` from 1.59 to 1.63 and the error went away. A little research shows that the feature was [marked stable as of 1.61](https://github.com/rust-lang/rust/pull/95491).

So perhaps we should indicate in `Cargo.toml` that the minimum Rust version is 1.61 so it can bail out early?

--- a/rinex/Cargo.toml
+++ b/rinex/Cargo.toml
@@ -9,7 +9,7 @@ keywords = ["rinex", "timing", "gps", "glonass", "galileo"]
 categories = ["science", "parsing"]
 edition = "2018"
 readme = "README.md"
-
+rust-version = "1.61"
 [features]
 # no features by default
 default = []

